ingestion/src/metadata/ingestion/source/database/database_service.py

```diff
@@ -231,11 +231,20 @@ class DatabaseServiceSource(DBTMixin, TopologyRunnerMixin, Source, ABC):
     context = create_source_context(topology)
 
     def __init__(self):
-        dbt_details = get_dbt_details(self.source_config.dbtConfigSource)
-        self.dbt_catalog = dbt_details[0] if dbt_details else None
-        self.dbt_manifest = dbt_details[1] if dbt_details else None
-        self.dbt_run_results = dbt_details[2] if dbt_details else None
-        self.data_models = {}
+        if hasattr(self.source_config.dbtConfigSource, "dbtSecurityConfig"):
```

Comment:
@MilanBariya this condition will only be true for GCS, S3 configs. In case of http, local configs `dbtSecurityConfig` is not present. Can you double check this logic here?